rust/integration-tests/src/diff_test_helper/processors/fungible_asset_processor.rs

```diff
@@ -0,0 +1,21 @@
+use crate::{
+    diff_test_helper::ProcessorTestHelper, models::queryable_models::FungibleAssetActivity,
+};
+use anyhow::Result;
+use diesel::{pg::PgConnection, query_dsl::methods::FilterDsl, ExpressionMethods, RunQueryDsl};
+use processor::schema::fungible_asset_activities::dsl::*;
+use serde_json::Value;
+
+pub struct FungibleAssetProcessorTestHelper;
+
+impl ProcessorTestHelper for FungibleAssetProcessorTestHelper {
+    fn load_data(&self, conn: &mut PgConnection, txn_version: &str) -> Result<Value> {
```

Comment:
do you know if the query result gets a different order? i wonder if that's going to break the test. if not can we force the order or order it before testing.

Reply:
unless the results are different, doesn't json_pretty re-order automatically? I can check. and if not, wecan force the ordered result when we load

Reply:
let's make sure the orders won't be a problem.

Reply:
oh in this case, only a single entry will be selected?